cmd/packages.go

```diff
@@ -101,14 +100,14 @@ func setPackageFlags(flags *pflag.FlagSet) {
 		"scope", "s", source.SquashedScope.String(),
 		fmt.Sprintf("selection of layers to catalog, options=%v", source.AllScopes))
 
-	flags.StringP(
-		"output", "o", string(format.TableOption),
-		fmt.Sprintf("report output formatter, options=%v", format.AllOptions),
+	flags.StringArrayP(
+		"output", "o", []string{string(format.TableOption)},
+		fmt.Sprintf("report output format, options=%v", format.AllOptions),
 	)
 
 	flags.StringP(
 		"file", "", "",
-		"file to write the report output to (default is STDOUT)",
+		"file to write the default report output to (default is STDOUT)",
```

Comment:
should this be deprecated?

Reply:
Is `-o` intended to replace `--file`?

Reply:
I don't really know, but the functionality is redundant. it would seem outputting a table to a file is fairly useless, so generally a user would have to do `-o spdx-json --file <output-file>`, for example. But this is already covered by the `-o spdx-json=<output-file>`...

Reply:
I see your point on redundancy. 

My take: I think it makes sense to leave the `--file` flag as it's not harming anything by leaving it. We probably don't need to add a deprecation notice if we aren't removing it (we could always decide to remove it in the future).